modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/onboarding/upgrade/UpgradePlusLayoutExperiment.kt

```diff
@@ -0,0 +1,185 @@
+                ) {
+                    Row(
+                        modifier = Modifier
+                            .fillMaxWidth()
+                            .padding(top = 8.dp),
+                        horizontalArrangement = Arrangement.End,
+                    ) {
+                        TextP30(
+                            text = stringResource(R.string.not_now),
+                            color = Color.White,
+                            modifier = Modifier
+                                .padding(start = 24.dp, end = 24.dp, top = 8.dp)
+                                .clickable { onNotNowPressed() },
+                        )
+                    }
+
+                    Column {
+                        Box(
+                            modifier = Modifier
+                                .fillMaxWidth()
+                                .padding(bottom = 12.dp, top = 24.dp),
+                            contentAlignment = Alignment.Center,
+                        ) {
+                            SubscriptionBadgeForTier(
+                                tier = SubscriptionTier.PLUS,
+                                displayMode = SubscriptionBadgeDisplayMode.ColoredWithBlackForeground,
+                                fontSize = 16.sp,
+                                padding = 8.dp,
+                            )
+                        }
+
+                        Box(
+                            modifier = Modifier.padding(bottom = 20.dp),
+                            contentAlignment = Alignment.Center,
+                        ) {
+                            AutoResizeText(
+                                text = stringResource(state.currentFeatureCard.titleRes(source)),
+                                color = Color.White,
+                                maxFontSize = 22.sp,
+                                lineHeight = 30.sp,
+                                fontWeight = FontWeight.W700,
+                                maxLines = 2,
+                                textAlign = TextAlign.Center,
+                                modifier = Modifier
+                                    .padding(horizontal = 24.dp)
+                                    .fillMaxWidth(),
+                            )
+                        }
+
+                        Spacer(Modifier.height(20.dp))
+
+                        FeatureCards( // TODO: it will be replaced for the new one
+                            state = state,
+                            upgradeButton = state.currentUpgradeButton,
+                            onFeatureCardChanged = onFeatureCardChanged,
+                        )
+                    }
+
+                    Spacer(Modifier.weight(1f))
+                }
+            }
+        }
+
+        if (canUpgrade) {
+            SubscribeButton(onClickSubscribe)
+        }
+    }
+}
+
+@Composable
+private fun SubscribeButton(
+    onClickSubscribe: () -> Unit,
+) {
```

Comment:
- Could this of reused the existing RowButton component?
- Or should we make this a reusable component?
- Can this have a preview added?

Reply:
I added a preview and changed to use this component